### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Megha-Joshi**
- Added success and error toast.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The PR is small and the code is clean and understandable
Good UI 
Good to merge

File: src/context/authContext.js [hunk 1]
```diff
@@ -24,8 +24,10 @@ const AuthProvider = ({children}) => {
                 console.log("in login handler")
```
Comment (zoyatanweer) [inline src/context/authContext.js:24]: remove console.log

File: src/context/videoContext.js [hunk 2]
```diff
@@ -106,10 +107,13 @@ const [ videoState, videoDispatch ] = useReducer(videoReducerFun, initialState);
                 console.log("entered like page");
```
Comment (zoyatanweer) [inline src/context/videoContext.js:107]: remove console.log

**Review by zoyatanweer — APPROVED** [review]
Code looks clean. Just do the required suggestions. 
Ui looks awesome.
 Good to merge.

File: src/context/authContext.js (hunk 1 above)
Comment (Megha-Joshi) [reply src/context/authContext.js:24]: Ok

File: src/context/videoContext.js (hunk 2 above)
Comment (Megha-Joshi) [reply src/context/videoContext.js:107]: Ok

**Pull request merged**